axiom/nr_distributed_trace.c

```diff
@@ -1197,11 +1221,19 @@ char* nr_distributed_trace_create_w3c_tracestate_header(
     sampled = "1";
   }
 
+  priority = nr_distributed_trace_get_priority(dt);
+
+  priority_buf = nr_priority_double_to_str(priority);
+  if (NULL == priority_buf) {
+    nrl_warning(NRL_CAT, "Failed to allocate priority buffer");
```

Comment:
Isn't warning too high of a log level here. There have been situations where excessive logging caused issues on customer systems. I'd suggest being more conservative:
```suggestion
    nrl_verbosedebug(NRL_CAT, "Failed to allocate priority buffer");
```

Reply:
In this case I would be concerned if this value was `NULL` since it would indicate that the call to `nr_formatf` in the `nr_priority_double_to_str` function failed. This should rarely (if ever) happen and I think it would warrant a higher log level for that reason.